### PR TITLE
xSqlServerNetwork example file

### PR DESCRIPTION
**Pull request by Arturas-K**
Hi,

As promised added example of new resource usage.
xSqlServerNetwork was added in #20;

### CODE REVIEW

File: Examples/SQLServerNetwork.ps1 [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            LocalSystemAccount = $LocalSystemAccount
+            AdminAccount = "CONTOSO\Administrator"
+        }
+        @{
+            NodeName = "nodename"
+            Instance = "MSSQLSERVER"
+            Features = "SQLENGINE"
+        } 
+    )
+}
```
Comment (TravisEz13) [inline Examples/SQLServerNetwork.ps1]: Files must end with a new line.  See [Style Guidelines, General Rule, #5](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#general-rules)

File: Examples/SQLServerNetwork.ps1 [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+$SecurePassword = ConvertTo-SecureString -String "Pass@word1" -AsPlainText -Force
+$InstallerServiceAccount = New-Object System.Management.Automation.PSCredential ("CONTOSO\!Installer", $SecurePassword)
+$LocalSystemAccount = New-Object System.Management.Automation.PSCredential ("SYSTEM", $SecurePassword)
+
+$ConfigurationData = @{
+    AllNodes = @(
+        #AllNodes
+        @{
+            NodeName = "*"
+            PSDscAllowPlainTextPassword = $true
```
Comment (TravisEz13) [inline Examples/SQLServerNetwork.ps1:69]: Add a comment that this is not recommended and point to a reference on how to do it correctly.  Such as the blog [want to secure credentials in windows PowerShell desired state configuration](http://blogs.msdn.com/b/powershell/archive/2014/01/31/want-to-secure-credentials-in-windows-powershell-desired-state-configuration.aspx)

File: Examples/SQLServerNetwork.ps1 [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+{
+    Import-DscResource -Module xSQLServer
+
+    # A Configuration expects at least one Node
+    Node $AllNodes.NodeName
+    {
+        # Set DCM Settings for each Node 
+        LocalConfigurationManager 
+        { 
+            DebugMode = "ForceModuleImport"
```
Comment (TravisEz13) [inline Examples/SQLServerNetwork.ps1]: DebugMode should only be enable for debugging.  It's ok to leave it here as a comment and say uncomment to debug.

**Comment by Arturas-K** [comment]
@TravisEz13 thank you for your comments.
I have made changes you requested.

File: Examples/SQLServerNetwork.ps1 (hunk 2 above)
Comment (TravisEz13) [inline Examples/SQLServerNetwork.ps1:69]: Verified done.

**Comment by TravisEz13** [comment]
Thanks for your contribution!

**Pull request merged**